### PR TITLE
Add ability to search by admin note when adding a question to a program

### LINKED ISSUE #4325: Allow admin search on admin note when adding a question
**Is your feature request related to a problem? Please describe.**
An admin may want to search on the string in an admin note when adding a question to a program, but right now that field isn't present

**Describe the solution you'd like**
Ability to include admin note in question filtering

### PR DESCRIPTION
**Pull request by dkatzz**
### Description

Admin note should be a field admins can search by when adding a question to a program

#### General

- [x] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release >
- [x] Created tests which fail without the change (if possible)

### Issue(s) this completes

Fixes #4325

### CODE REVIEW

**Comment by bion** [comment]
Please clean up PR description

File: browser-test/src/admin_program_creation.test.ts [hunk 1]
```diff
@@ -37,8 +37,30 @@ describe('program creation', () => {
       'program-creation-static-question-with-formatting',
     )
   })
+  it('create program and search for questions', async () => {
```
Comment (bion) [inline browser-test/src/admin_program_creation.test.ts:40]: **nitpick:** space between test cases
```suggestion

  it('create program and search for questions', async () => {
```

File: browser-test/src/admin_program_creation.test.ts [hunk 2]
```diff
@@ -37,8 +37,30 @@ describe('program creation', () => {
...
+    )
+    expect(await page.innerText('id=question-bank-questions')).toContain(
+      'this is a note',
+    )
+
+    await validateScreenshot(page, 'open-question-search')
+  })
 
-  it('create program with enumerator and repeated questions', async () => {
+    it('create program with enumerator and repeated questions', async () => {
```
Comment (bion) [inline browser-test/src/admin_program_creation.test.ts:63]: **nitpick:** revert

**Review by bion — APPROVED** [review]

File: browser-test/src/admin_program_creation.test.ts (hunk 1 above)
Comment (dkatzz) [reply browser-test/src/admin_program_creation.test.ts:41]: Sorry about that, updated!

File: browser-test/src/admin_program_creation.test.ts (hunk 2 above)
Comment (dkatzz) [reply browser-test/src/admin_program_creation.test.ts]: I hadn't pushed my fmt changes at first - sorry! Updated.

**Pull request merged**